cmd/dao_tests/pubkey.go

```diff
@@ -0,0 +1,208 @@
+package main
+
+import (
+	"context"
+	"testing"
+
+	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	_ "github.com/RHEnVision/provisioning-backend/internal/dao/sqlx"
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/RHEnVision/provisioning-backend/internal/config"
+	"github.com/RHEnVision/provisioning-backend/internal/db"
+)
+
+func createPk() []*models.Pubkey {
+	return []*models.Pubkey{{
+		ID:        10,
+		AccountID: 2,
+		Name:      "testkey",
+		Body:      "sha-rsa body",
+	}}
+}
+
+func updatePk() []*models.Pubkey {
+	return []*models.Pubkey{{
+		ID:        1,
+		AccountID: 1,
+		Name:      "updated-key",
+		Body:      "updated body",
+	}}
+}
+
+func TearDown(t *testing.T) {
```

Comment:
We call this TearDown at the beggining of every test, I'd love if we could use some wrapper for all tests that would make sure TearDown happens after the run of every test and Setup is called prior the run of every test.